File: Modul 5/C/PRAK504.c
#include <stdio.h>

int reverse(int bilangan){
// Lengkapi Function ini
    int reverse = 0;
    int sisa = 0;
    while (bilangan != 0) {
        sisa = bilangan % 10;
        reverse = reverse * 10;
        reverse = reverse + sisa;
        bilangan = bilangan / 10;
    }
    return reverse;
}
int main() {
 int A, B;
 scanf("%d %d",&A,&B);
 A=reverse(A);
 B=reverse(B);
 int C = A+B;
 printf("%d",reverse(C));
}

File: Modul 5/Python/PRAK504.py
def reverse(bilangan):
    reverse = 0
    sisa = 0
    while (bilangan != 0):
        sisa = bilangan % 10;
        reverse = reverse * 10;
        reverse = reverse + sisa;
        bilangan = bilangan // 10;
    return reverse;

angka = input().split()
A, B = map(int,angka)
A_reverse = reverse(A)
B_reverse = reverse(B)
C = A_reverse + B_reverse
print(reverse(C))